core/services/offchainreporting/oracle.go

```diff
@@ -115,7 +115,18 @@ func (d jobSpawnerDelegate) ServicesForSpec(spec job.Spec) ([]job.Service, error
 		return nil, errors.Wrap(err, "could not make new peerstore")
 	}
 
-	ocrLogger := NewLogger(logger.Default, d.config.OCRTraceLogging())
+	var jb models.JobSpecV2
+	err = d.db.First(&jb, "id = ?", spec.JobID()).Error
```

Comment:
Can we reduce this query to only fetch the pipeline spec ID, since that's all we need?

Reply:
Hmm I'm not sure I like the alternative better: 
```
	var pipelineSpecID []int32
	err = d.db.Table("jobs").
		Where("id = ?", spec.JobID()).
		Pluck("pipeline_spec_id", &pipelineSpecID).
		Error
	if err != nil {
		return nil, errors.Wrapf(err, "could not find job with ID %d", spec.JobID())
	}
        // awkwardly use pipelineSpecID[0] ?
```

Reply:
given its a 3 column table I think its actually better as is

Reply:
Yup I agree